src/main/java/com/jalasoft/jfc/model/ImageConverter.java

```diff
@@ -36,20 +38,21 @@ protected boolean convertImage(ImageParam image) {
         verifyDataValues(image);
         boolean returnTrueOrFalse = false;
```

Comment:
you can improve varible name "returnTrueOrFalse".
for example to "resultAccion"